src/app.js

```diff
@@ -26,10 +37,40 @@ const quoteData = [
 ];
 
 $(document).ready(function() {
+  let bus = {};
+  bus = _.extend(bus, Backbone.Events);
+  tradeTemplate = _.template($('#trade-template').html());
+  quoteTemplate = _.template($('#quote-template').html());
+  orderTemplate = _.template($('#order-template').html());
   const quotes = new QuoteList(quoteData);
+
+  const orders = new OrderList(); // not sure if this is right...
```

Comment:
It looks OK to me. The list of limit orders starts empty, so it makes sense that the collection would be created without any starter data.